docs/example/multiqc_data/mqc_number_of_peptides_per_proteins__name_percentage_ylab_Percentage_.txt

```diff
@@ -0,0 +1,26 @@
+Sample	Percentage
```

Comment:
Sorting is not properly working. 

- [ ] When sorting the Peptide Quant table by `BestSearchScore` the first peptide score is not the highest one.
- [ ] After sorting the first time in the BestSearchScore, the users can't do any more sorting in any other column. I have tested the Average Intensity.  
 
![Screen Shot 2022-06-06 at 08 53 55](https://user-images.githubusercontent.com/52113/172123037-339370d9-a05f-467e-88f3-e0949cfda666.png)

I need to said that this is also happening for the Protein Table.